ci/script.sh

```diff
@@ -8,6 +8,15 @@ set -ex
 cargo build --verbose
 cargo doc --verbose
 
+# If we're testing on an older version of Rust, then only check that we
+# can build the crate. This is because the dev dependencies might be updated
+# more frequently, and therefore might require a newer version of Rust.
+#
+# This isn't ideal. It's a compromise.
```

Comment:
An alternative compromise that has been suggested is to test MRV with `minimal-versions` so that it does not use the updated dependencies. This involves using a nightly to build the lockfile then using the MRV to build the result. It is not a silver bullet, but it is another option.

Reply:
Neat idea, but I think I'll pass unless something comes up that motivates it. :-)